code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1199,6 +1199,146 @@ func spStats(
 //	400:
 //	500:
 func (srh *StorageRestHandler) getStakePoolStat(w http.ResponseWriter, r *http.Request) {
+	blobberID := r.URL.Query().Get("blobber_id")
+	if blobberID != "" {
+		srh.getBlobberStakePoolStat(w, r)
+		return
+	}
```

Comment:
The blobber_id param is not needed if we use provider_id. Please also update gosdk, zwalletcli, zboxcli, and system tests if needed for this API change.